persistence-modules/core-java-persistence/README.md

```diff
@@ -10,3 +10,4 @@
 - [Guide to the JDBC ResultSet Interface](https://www.baeldung.com/jdbc-resultset)
 - [Types of SQL Joins](https://www.baeldung.com/sql-joins)
 - [Returning the Generated Keys in JDBC](https://www.baeldung.com/jdbc-returning-generated-keys)
+- [Difference between Statement and PreparedStatement](https://www.baeldung.com/difference-between-statement-and-preparedstatement)
```

Comment:
Do not edit this file. We don't know what the final URL will be yet, and your editor will add this after the article is published.